### PR TITLE
FAQ Refactor + Support for anchor string in text + English product catalog

### PR DESCRIPTION
**Pull request by rikaznismi**

### CODE REVIEW

File: src/pages/faq.tsx [hunk 1]
```diff
@@ -1,104 +1,52 @@
...
+            <a key={i} href={href} className="text-cerise underline">
+              {part.replace(/<a[^>]*>|<\/a>/g, "")}
+            </a>
+          );
+        } else {
+          return <span>{part}</span>;
+        }
+      })}
+    </p>
+  );
```
Comment (ernum) [inline src/pages/faq.tsx:22]: Nice!

File: src/pages/faq.tsx [hunk 2]
```diff
@@ -1,104 +1,52 @@
...
 }
```
Comment (ernum) [inline src/pages/faq.tsx:23]: Nice!

File: src/pages/faq.tsx [hunk 3]
```diff
@@ -1,104 +1,52 @@
...
-
-    <button onClick={() => stateAction[0][1](!stateAction[0][0])} id="accordion" className="pl-[20px] text-left text-white h-[130px] sm:h-[55px] w-full bg-slate-50 bg-opacity-20 border-[3px] border-cerise"> {t.faq.table4row1}</button>
-    {stateAction[0][0] ? (
-    <div className="pl-[20px] h-[350px] sm:h-[100px] bg-gray bg-opacity-50 border-[1px] border-cerise flex items-center ">
-    <p className="text-white">{t.faq.table4text1} <a className="text-cerise" href="mailto:alexandre.moch@ddagen.se">alexandre.moch@ddagen.se</a></p>
-    </div>) : null}
+    <div className="mt-[50px]">
+
+    {questions.map((_, i) => (
+      <div>
```
Comment (ernum) [inline src/pages/faq.tsx:32]: We get the following warning right now: 

```
Warning: Each child in a list should have a unique "key" prop.

Check the render method of `Faq`. See https://reactjs.org/link/warning-keys for more information.
    at div
    at Faq (webpack-internal:///./src/pages/faq.tsx:94:66)
    at div
    at App (webpack-internal:///./src/pages/_app.tsx:19:11)
    at PathnameContextProviderAdapter (webpack-internal:///./node_modules/next/dist/shared/lib/router/adapters.js:62:11)
    at ErrorBoundary (webpack-internal:///./node_modules/next/dist/compiled/@next/react-dev-overlay/dist/client.js:301:63)
    at ReactDevOverlay (webpack-internal:///./node_modules/next/dist/compiled/@next/react-dev-overlay/dist/client.js:850:919)
    at Container (webpack-internal:///./node_modules/next/dist/client/index.js:62:1)
    at AppContainer (webpack-internal:///./node_modules/next/dist/client/index.js:197:11)
    at Root (webpack-internal:///./node_modules/next/dist/client/index.js:372:11)
```

To fix this we can simply add `key={i}` in this div.

File: src/pages/faq.tsx [hunk 4]
```diff
@@ -1,104 +1,52 @@
 import { useLocale } from "@/locales";
```
Comment (ernum) [inline src/pages/faq.tsx:1]: I think we should format the document with `prettier`.

File: src/pages/faq.tsx (hunk 3 above)
Comment (rikaznismi) [reply src/pages/faq.tsx:32]: Nice, I will fix this.

File: src/pages/faq.tsx [hunk 5]
```diff
@@ -185,14 +136,16 @@ export default function Faq() {
...
       
       {/*Product Catalog button*/}
-      <button className="mt-[100px] h-[100px] w-[250px] bg-cerise rounded-[40px] border-cerise flex items-center justify-center px-[30px] flex-col">
-        <a href="/downloadables/Product_Catalog.pdf" download="Product_Catalog" className="text-white text-[22px] text-center">{t.faq.download}</a>
-        <div className="flex items-center">
-          <span className="text-white text-[20px]">{t.faq.productCatalog}</span>
-        </div>
-      </button>
+      <a href={t.faq.catalogPath}>
+        <button className="mt-[100px] h-[100px] w-[250px] bg-cerise rounded-[40px] border-cerise flex items-center justify-center px-[30px] flex-col">
```
Comment (ernum) [inline src/pages/faq.tsx:140]: Maybe we should change this to simply `Product Catalog` since they are no longer download it?

Comment (rikaznismi) [reply src/pages/faq.tsx]: Fixed

File: src/pages/faq.tsx (hunk 3 above)
Comment (rikaznismi) [reply src/pages/faq.tsx]: Fixed

**Pull request merged**